src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -1732,6 +1735,92 @@ public ResponseEntity<Void> invalidateShortCircuitStatus(@Parameter(description
+                                                              @Parameter(description = "nodeUuid") @PathVariable("nodeUuid") UUID nodeUuid) {
+        String result = nonEvacuatedEnergyService.getNonEvacuatedEnergyResult(nodeUuid);
+        return result != null ? ResponseEntity.ok().body(result) :
+            ResponseEntity.noContent().build();
+    }
+
+    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/sensitivity-analysis-non-evacuated-energy/status")
+    @Operation(summary = "Get the sensitivity analysis non evacuated energy status on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The sensitivity analysis non evacuated energy status"),
+        @ApiResponse(responseCode = "204", description = "No sensitivity analysis non evacuated energy has been done yet"),
+        @ApiResponse(responseCode = "404", description = "The sensitivity analysis status non evacuated energy has not been found")})
+    public ResponseEntity<String> getNonEvacuatedEnergyStatus(@Parameter(description = "Study UUID") @PathVariable("studyUuid") UUID studyUuid,
+                                                              @Parameter(description = "nodeUuid") @PathVariable("nodeUuid") UUID nodeUuid) {
+        String result = nonEvacuatedEnergyService.getNonEvacuatedEnergyStatus(nodeUuid);
+        return result != null ? ResponseEntity.ok().body(result) :
+            ResponseEntity.noContent().build();
+    }
+
+    @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/sensitivity-analysis-non-evacuated-energy/stop")
+    @Operation(summary = "stop sensitivity analysis non evacuated energy on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The sensitivity analysis non evacuated energy has been stopped")})
+    public ResponseEntity<Void> stopNonEvacuatedEnergy(@Parameter(description = "Study uuid") @PathVariable("studyUuid") UUID studyUuid,
+                                                       @Parameter(description = "nodeUuid") @PathVariable("nodeUuid") UUID nodeUuid) {
+        nonEvacuatedEnergyService.stopNonEvacuatedEnergy(studyUuid, nodeUuid);
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping(value = "/studies/{studyUuid}/sensitivity-analysis-non-evacuated-energy/parameters")
+    @Operation(summary = "Get sensitivity analysis non evacuated energy parameters on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The sensitivity analysis non evacuated energy parameters")})
+    public ResponseEntity<org.gridsuite.study.server.dto.sensianalysis.nonevacuatedenergy.NonEvacuatedEnergyParametersInfos> getNonEvacuatedEnergyParametersValues(
+        @PathVariable("studyUuid") UUID studyUuid) {
+        return ResponseEntity.ok().body(studyService.getNonEvacuatedEnergyParametersValues(studyUuid));
+    }
+
+    @PostMapping(value = "/studies/{studyUuid}/sensitivity-analysis-non-evacuated-energy/parameters")
```

Comment:
why some are sensitivity-analysis-non-evacuated-energy and other only non-evacuated-energy? I think the latter would be more consistent ?

Reply:
Renamed